app.js

```diff
@@ -1,13 +1,33 @@
 const express = require("express");
-const { getController } = require("./controllers/controller");
+const { getController, getReviewById } = require("./controllers/controller");
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/categories", getController);
 
-app.use("*", (req, res) => {
+app.get("/api/reviews/:review_id", getReviewById);
+
+// ---ERRORS---
+
+app.all("*", (req, res) => {
   res.status(404).send({ msg: "Route not found!" });
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "review_id must be a number" });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err) {
+    res.status(err.status).send({ msg: err.msg });
```

Comment:
You should be more specific in the condition here as the next error handler will never be reached